install_config/persistent_storage/topics/glusterfs_example_full.adoc

```diff
@@ -3,20 +3,37 @@
 ----
 [OSEv3:vars]
 ...
-openshift_registry_selector="role=infra" <1>
+openshift_registry_selector="role=infra"                  <1>
 openshift_hosted_registry_storage_kind=glusterfs
 
+openshift_metrics_hawkular_nodeselector={"role":"infra"}  <1>
+openshift_metrics_cassandra_nodeselector={"role":"infra"} <1>
+openshift_metrics_heapster_nodeselector={"role":"infra"}  <1>
+openshift_metrics_storage_kind=dynamic
+openshift_metrics_cassanda_pvc_storage_class_name="glusterfs-registry-block" <2>
+
+openshift_logging_es_nodeselector={"role":"infra"}        <1>
+openshift_logging_kibana_nodeselector={"role":"infra"}    <1>
+openshift_logging_curator_nodeselector={"role":"infra"}   <1>
+openshift_logging_storage_kind=dynamic
+openshift_logging_es_pvc_size=10Gi                        <3>
+openshift_logging_es_pvc_storage_class_name="glusterfs-registry-block"       <2>
+
 openshift_storage_glusterfs_block_deploy=false
```

Comment:
Is this still needed if both gluster storageclasses are not default anymore?

Reply:
The default is true, and we don't want to deploy gluster-block functionality for the non-infra cluster, so yes.